src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -181,66 +174,59 @@ function addScriptTagsToDocument(resourceLoader: WebAssemblyResourceLoader) {
     .keys(resourceLoader.bootConfig.resources.runtime)
     .filter(n => n.startsWith('dotnet.') && n.endsWith('.js'))[0];
   const dotnetJsContentHash = resourceLoader.bootConfig.resources.runtime[dotnetJsResourceName];
-  const scriptElem = document.createElement('script');
-  scriptElem.src = `_framework/${dotnetJsResourceName}`;
-  scriptElem.defer = true;
-
-  // For consistency with WebAssemblyResourceLoader, we only enforce SRI if caching is allowed
-  if (resourceLoader.bootConfig.cacheBootResources) {
-    scriptElem.integrity = dotnetJsContentHash;
-    scriptElem.crossOrigin = 'anonymous';
-  }
+  let src = `./${dotnetJsResourceName}`;
+
+  // TODO How to do SRI dotnetJsContentHash with dynamic import ?
```

Comment:
Is this something we need to address before merging?

Reply:
Not from my perspective, but I don't know what impact it has downstream.

Reply:
Discussed offline with @pavelsavara, we don't believe this'll be necessary.

```suggestion
```